File: AccountService.py
class AccountService:
    @staticmethod
    def find_user_by_username(username, users):
        for user in users:
            if user.username == username:
                return user
        return None

    @staticmethod
    def view_personal_information(user):
        print("Personal Information:")
        print("Username: ", user.username)
        print("First Name: ", user.first_name)
        print("Last Name: ", user.last_name)
        print("Email: ", user.email)
        print("Phone Number: ", user.phone_number)

    @staticmethod
    def edit_personal_information(user):
        print("Enter new Last Name:")
        last_name = input()
        print("Enter new First Name:")
        first_name = input()
        print("Enter new Email:")
        email = input()
        print("Enter new Phone Number:")
        phone_number = input()

        user.last_name = last_name
        user.first_name = first_name
        user.email = email
        user.phone_number = phone_number

        AccountService.view_personal_information(user)

    @staticmethod
    def change_password(user):
        print("Enter current password:")
        current_password = input()
        print("Enter new password:")
        new_password = input()
        print("Repeat new password:")
        repeated_password = input()

        if current_password == user.password and new_password == repeated_password:
            user.password = new_password
            print("Password updated")
        else:
            print("Wrong password")

File: AccountService.js
class AccountService {
    static findUserByUsername(username, users) {
        return users.find(user => user.username === username);
    }

    static viewPersonalInformation(user) {
        console.log("Personal Information:");
        console.log("Username:", user.username);
        console.log("First Name:", user.firstName);
        console.log("Last Name:", user.lastName);
        console.log("Email:", user.email);
        console.log("Phone Number:", user.phoneNumber);
    }

    static editPersonalInformation(user) {
        console.log("Enter new Last Name:");
        const lastName = prompt();
        console.log("Enter new First Name:");
        const firstName = prompt();
        console.log("Enter new Email:");
        const email = prompt();
        console.log("Enter new PhoneNumber:");
        const phoneNumber = prompt();

        user.lastName = lastName;
        user.firstName = firstName;
        user.email = email;
        user.phoneNumber = phoneNumber;

        AccountService.viewPersonalInformation(user);
    }

    static changePassword(user) {
        console.log("Enter current password:");
        const currentPassword = prompt();
        console.log("Enter new password:");
        const newPassword = prompt();
        console.log("Repeat new password:");
        const repeatedPassword = prompt();

        if (currentPassword === user.password && newPassword === repeatedPassword) {
            user.password = newPassword;
            console.log("Password updated");
        } else {
            console.log("Wrong password");
        }
    }
}

module.exports = { AccountService };
